File: P6.SumSquareDifference/Java/SumSquareDifference.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class SumSquareDifference {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for(int a0 = 0; a0 < t; a0++){
            int n = in.nextInt();
            long sum = 0;
            long sqrSum = 0;
            for(int i = 1;i <= n; i++){
                sum += i;
                sqrSum += (long) Math.pow(i,2);
            }
            
            sum = (long) Math.pow(sum,2);
            sum -= sqrSum;
            
            System.out.println(sum);
        }
    }
}

File: P6.SumSquareDifference/C/sum-square-difference.c
#include <math.h>
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <assert.h>
#include <limits.h>
#include <stdbool.h>

int main(){
    int t; 
    scanf("%d",&t);
    for(int a0 = 0; a0 < t; a0++){
        int n; 
        scanf("%d",&n);
        long long sum = 0;
        long long sqrSum = 0;
        
        for(int i = 1;i <= n; i++){
            sum += i;
            sqrSum += pow(i, 2);
        }
        
        sum = pow(sum, 2);
        sum -= sqrSum;
        
        printf("%lld\n", sum);
    }
    return 0;
}